data/json/recipes/food/meat.json

```
@@ -2544,5 +2544,132 @@
     "qualities": [ { "id": "COOK", "level": 2 } ],
     "tools": [ [ [ "frying_oil", 1, "LIST" ] ], [ [ "surface_heat", 8, "LIST" ] ] ],
     "components": [ [ [ "offal_raw", 1, "LIST" ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "sandwich_okay",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "2 m 30 s",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [
+      [ [ "meat_cooked", 1, "LIST" ] ],
+      [ [ "bread_sandwich", 2, "LIST" ] ],
+      [
+        [ "veggy_cooked", 1 ],
+        [ "veggy_wild_cooked", 1 ],
+        [ "veggy_salted", 1 ],
+        [ "celery", 1 ],
+        [ "irradiated_celery", 1 ],
+        [ "cabbage", 1 ],
+        [ "irradiated_cabbage", 1 ],
+        [ "sauerkraut", 1 ],
+        [ "tomato", 1 ],
+        [ "can_tomato", 1 ],
+        [ "irradiated_pumpkin", 1 ],
+        [ "onion", 1 ],
+        [ "irradiated_onion", 1 ],
+        [ "broccoli", 1 ],
+        [ "irradiated_tomato", 1 ],
+        [ "irradiated_broccoli", 1 ],
+        [ "rehydrated_veggy", 1 ],
+        [ "morel_cooked", 1 ],
+        [ "mushroom_cooked", 1 ],
+        [ "sauerkraut_onions", 1 ],
+        [ "pickle", 1 ],
+        [ "veggy_canned", 1 ],
+        [ "beans_cooked", 1 ],
+        [ "irradiated_carrot", 1 ],
+        [ "irradiated_zucchini", 1 ],
+        [ "irradiated_lettuce", 1 ],
+        [ "carrot", 1 ],
+        [ "zucchini", 1 ],
+        [ "lettuce", 1 ],
+        [ "pine_nuts", 1 ],
+        [ "morel_fried", 1 ],
+        [ "veggy_aspic", 1 ],
+        [ "dahlia_baked", 1 ],
+        [ "irradiated_cucumber", 1 ],
+        [ "cucumber", 1 ],
+        [ "veggy_pickled", 1 ],
+        [ "cooked_cattail_stalk", 1 ],
+        [ "cooked_pumpkin", 1 ],
+        [ "potato_baked", 1 ],
+        [ "irradiated_potato", 1 ],
+        [ "can_tomato", 1 ],
+        [ "dandelion_cooked", 1 ],
+        [ "burdock_cooked", 1 ],
+        [ "chili_pepper_roasted", 1 ],
+        [ "spinach", 1 ],
+        [ "grape_leaves", 1 ]
+      ]
```

could we use requirement list, `"veggy_any"` to simplify this component? (please don't apply this suggestion as a commit, it might mess up with formatting)
```suggestion
        [ [ "veggy_any", 1, "LIST" ] ]
```